### PR TITLE
Update a bunch of changelog entries.

### PR DESCRIPTION
**Pull request by bangerth**
This updates a bunch of changelog entries added last week.

### CODE REVIEW

File: doc/news/changes/minor/20190807Gleeson [hunk 1]
```diff
@@ -1,3 +1,3 @@
-Changed: The step-4 tutorial commented description of the right-hand side function and importance of dim, has been updated for grammatical clarity.   
+Changed: The step-4 tutorial commented description of the right-hand side function and importance of dim have been updated for grammatical clarity.
```
Comment (kronbichler) [inline doc/news/changes/minor/20190807Gleeson]: Would you mind breaking the line at 80 characters?

File: doc/news/changes/minor/20190807ManaswineeBezbaruah [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Improved: Changes in grid_generator.h
+Improved: Changes in namespace GridGenerator.
 The original documentation of enclosed_hyper_cube said 'first two parameters' in the second line, which was a bit unclear.
```
Comment (kronbichler) [inline doc/news/changes/minor/20190807ManaswineeBezbaruah]: Can you wrap lines at 80 characters (despite pre-existing)?

File: doc/news/changes/minor/20190808KatherineCosburn [hunk 3]
```diff
@@ -1,3 +1,3 @@
-Found the original wording asking for user to explore more refinements to be confusing and redundant given that previous paragraph asks to look at results of a p-refinement. Assuming they meant to now explore h-refinement, I changed the text to make this more clear.
-
+Changed: Found the original wording asking for user to explore more refinements to be confusing and redundant given that previous paragraph asks to look at results of a p-refinement. Assuming they meant to now explore h-refinement, I changed the text to make this more clear.
```
Comment (kronbichler) [inline doc/news/changes/minor/20190808KatherineCosburn]: same here

**Comment by tjhei** [comment]
I think we agreed to merge all the "fixed a typo" changelog entries into a single one. Do you want to do this instead of fixing the formatting of every single entry here?

**Comment by bangerth** [comment]
I think this has been superseded now.

**Pull request closed without merging**